packages/desktop/src/app.rs

```diff
@@ -134,6 +138,29 @@ impl App {
             _ => (),
         }
     }
+    #[cfg(any(target_os = "windows", target_os = "linux", target_os = "macos"))]
+    pub fn handle_tray_menu_event(&mut self, event: tray_icon::menu::MenuEvent) {
```

Comment:
What's happening here? Should we be firing any handlers?

Reply:
The thing is it could be the same as https://github.com/DioxusLabs/dioxus/blob/733d16236986d778c2951defcb939a9f97ac1147/packages/desktop/src/app.rs#L115-L140 
but I don't know how useful that would be to open dev tools and make the window float on top from the tray instead of the menu so if there is some event that would make sense to be in the tray let me know, otherwise I can remove that.